engine/crates/engine/src/schema.rs

```diff
@@ -566,9 +577,7 @@ impl Schema {
                             .execute(env.operation_name.as_deref(), &env.operation, &mut fut)
                             .await
                     }
-                    // here we don't know the type of the operation because it failed preparing the request
-                    // defaulting but this might not be the best option
-                    Err(errors) => Response::from_errors(errors, Default::default()),
+                    Err(errors) => Response::bad_request(errors),
```

Comment:
Was the Default impl on OperationType for this call site? Is it still necessary?

Reply:
Good catch!